book_source/02_demos_tutorials_workflows/05_developer_workflows/02_git/01_using-git.Rmd

```diff
@@ -106,41 +119,9 @@ cd pecan
 git remote add upstream git@github.com:PecanProject/pecan.git
 ```
 
-#### During development:
-
-* commit often; 
-* each commit can address 0 or 1 issue; many commits can reference an issue
-* ensure that all tests are passing before anything is pushed into develop.
-
-#### Basic Workflow
-
-This workflow is for educational purposes only. Please use the Recommended Workflow if you plan on contributing to PEcAn. This workflow does not include creating branches, a feature we would like you to use.
-1. Get the latest code from the main repository
-
-`git pull upstream develop`
-
-2. Do some coding
-
-3. Commit after each chunk of code (multiple times a day)
-
-`git commit -m "<some descriptive information about what was done; references/fixes gh-X>"`
-
-4. Push to YOUR Github (when a feature is working, a set of bugs are fixed, or you need to share progress with others)
-
-`git push origin develop`
-
-5. Before submitting code back to the main repository, make sure that code compiles from the main directory.
-
-`make`
-
-
-6. submit pull request with a reference to related issue; 
-* also see [github documentation](https://help.github.com/articles/using-pull-requests)
-
-
```

Comment:
👍 👍 to removing this. It's pretty confusing to lead off with a workflow we don't recommend using.